fixtures/emails/digest.txt

```diff
@@ -1,55 +1,37 @@
 Notifications for example
```

Comment:
this fixture file changed because I removed a dead `random` call which caused the seeding (and results) to be different